api/v1beta1/module_types.go

```diff
@@ -165,6 +165,11 @@ type ModuleLoaderContainerSpec struct {
 	// Sign provides default kmod signing settings
 	Sign *Sign `json:"sign,omitempty"`
 
+	// KernelModuleVersion defines the current version of the kernel module being used
+	// Used for upgrading the currently loaded kernel module to a new version
+	// +kubebuilder:default="1.0"
+	KernelModuleVersion string `json:"kernelModuleVersion"`
```

Comment:
Why `kernelModuleVersion` and not just `moduleVersion`? Or even `version`?

Reply:
I am opened to whatever naming you deem suitable. Initially i named it ModuleVersion, but then i thought that KernelModuleVersion is more accurate.

Reply:
I like the `version` suggestions as well.

Reply:
fixed